Demo/Application/Features/VenmoViewController.swift

```diff
@@ -7,26 +7,33 @@ class VenmoViewController: PaymentButtonBaseViewController {
     var venmoClient: BTVenmoClient!
 
     let vaultToggle = Toggle(title: "Vault")
-    let universalLinkReturnToggle = Toggle(title: "Use Universal Link Return")
 
     override func viewDidLoad() {
         super.heightConstraint = 150
         super.viewDidLoad()
-        venmoClient = BTVenmoClient(apiClient: apiClient)
+        venmoClient = BTVenmoClient(
+            apiClient: apiClient,
+            // swiftlint:disable:next force_unwrapping
+            universalLink: URL(string: "https://mobile-sdk-demo-site-838cead5d3ab.herokuapp.com/braintree-payments")!
+        )
+
         title = "Custom Venmo Button"
     }
     
     override func createPaymentButton() -> UIView {
         let venmoButton = createButton(title: "Venmo", action: #selector(tappedVenmo))
 
-        let stackView = UIStackView(arrangedSubviews: [vaultToggle, universalLinkReturnToggle, venmoButton])
-        stackView.axis = .vertical
-        stackView.spacing = 15
-        stackView.alignment = .fill
-        stackView.distribution = .fillEqually
-        stackView.translatesAutoresizingMaskIntoConstraints = false
+        let venmoStackView = buttonsStackView(label: "Venmo Payment Flow", views: [
+            vaultToggle,
+            venmoButton
+        ])
+
+        venmoStackView.axis = .vertical
+        venmoStackView.distribution = .fillProportionally
```

Comment:
I think some of these properties are being set within the same method. Maybe we can add the missing ones

Reply:
great catch, updated: https://github.com/braintree/braintree_ios/pull/1489/commits/1debdb9b39897dc37b3c9d7a5b2f098600152ce1